docs/ARCHITECTURE.md

```
@@ -4,6 +4,46 @@ High-level architecture overview of the Lucid Agents SDK.
+```
+
+**@lucid-agents/api-sdk** enables:
+- **Agent Factories**: Build agents that create and manage other agents
+- **Platform Dashboards**: Build UIs for agent management and analytics
+- **Agent-to-Agent**: Discover and invoke other agents with x402 payments
+
+See [API-SDK Documentation](./API-SDK.md) for full details.
+
+---
+
+## Package Structure
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)